### PR TITLE
Oxygen orbs sound rework

### PR DESCRIPTION
**Pull request by 6819710**
Reworked to match other work done, and allow for ballancing / editing without changing underlying audio tracks

### CODE REVIEW

File: Assets/Scripts/Audio/SoundEffectsController.cs [hunk 1]
```diff
@@ -1,38 +0,0 @@
-﻿using System.Collections;
-using System.Collections.Generic;
-using UnityEngine;
-
-[RequireComponent(typeof(AudioSource))]
-public class SoundEffectsController : MonoBehaviour {
```
Comment (xxfast) [inline Assets/Scripts/Audio/SoundEffectsController.cs]: There's other game objects in the scene that utilise this script. Don't delete it, just keep it in the scripts folder if you don't want to use what i've already written.

File: Assets/Scripts/GrantOxygenOnCollect.cs [hunk 2]
```diff
@@ -35,12 +35,8 @@ public class GrantOxygenOnCollect : MonoBehaviour {
 					oxygen.ApplyDelta (grantingRate);
 				}
 			}
-			//Trigger Soundeffects , If the reciever has an audio source
-			SoundEffectsController sfx = collision.gameObject.GetComponent<SoundEffectsController> ();
-			if (sfx != null && sfx is PlayerSFXController) { // if it's players
-				PlayerSFXController psfx = sfx as PlayerSFXController;
-				psfx.Play (PlayerSFXController.SoundEffect.Oxygen);
```
Comment (xxfast) [inline Assets/Scripts/GrantOxygenOnCollect.cs]: When a new sound effect triggered, it cuts off the previous one. This - in my opinion - not intended and kinda sounds weird

**Comment by 6819710** [comment]
[Oxygen Sucker SFX trigger Rework](https://trello.com/c/TBXu262X/34-oxygen-sucker-sfx-trigger-rework)

**Comment by 6819710** [comment]
Simpler just to re allign as part of task than deal with merge conflicts.

**Pull request closed without merging**